Src/BootCamp.Chapter/Demo.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    internal class Demo1
+    {
+        internal static void TwoDGrid()
+        {
+            Console.WriteLine("How much rows do you want the grid to be");
+            var input = Console.ReadLine();
+            var rows = ValidateInput(input);
+
+            var grid = new bool[rows, rows];
+            for (int i = 0; i < rows; i++)
```

Comment:
For loop can be removed. New array will have all the default values of the type it is for. In this case default value of bool is false either way.

Reply:
oke, done